app/views/api/integrations/apicast/shared/_proxy_endpoints.html.slim

```diff
@@ -9,11 +9,9 @@
                           placeholder: "https://api.#{parameterized_org_name_of_the_current_account}.com" },
             hint: apicast_endpoint_input_hint(service, environment: 'staging')
 
-  = help_bubble( 'proxy_endpoint_bubble', t('api.integrations.proxy.proxy_endpoint_help_html'))
```

Comment:
We still use the locale `proxy_endpoint_help_html`? if not, which seems so, then we should remove it:
https://github.com/3scale/porta/blob/7ddf357807fcca2400c674bdf7c7ad0a1aa364d7/config/locales/en.yml#L514-L517

Same for the rest of locales related to these bubbles.

Reply:
You are right, removed!